spring-rabbit/src/test/java/org/springframework/amqp/rabbit/annotation/EnableRabbitIntegrationTests.java

```diff
@@ -306,6 +320,56 @@ public RabbitAdmin rabbitAdmin(ConnectionFactory connectionFactory) {
 			return new RabbitAdmin(connectionFactory);
 		}
 
+		@Bean
+		public MultiListenerBean multiListener() {
+			return new MultiListenerBean();
+		}
+
+	}
+
+	@RabbitListener(bindings = @QueueBinding
+			(value = @Queue,
```

Comment:
Having this I'd say that `@QueueBinding` can be fixed like:
```
Queue value() default @Queue;
```
Which means that we really use an `AnonymousQueue`